src/utils/migrations/0035_rorimport_rorimporterror.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 4.2.14 on 2024-07-26 20:51
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('utils', '0038_upgrade_1_7_2'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='RORImport',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('started', models.DateTimeField(auto_now_add=True)),
+                ('stopped', models.DateTimeField(blank=True, null=True)),
+                ('status', models.CharField(choices=[('ongoing', 'Ongoing'), ('unnecessary', 'Unnecessary'), ('successful', 'Successful'), ('failed', 'Failed')], default='ongoing')),
```

Comment:
Should these choices reference RORImportStatus rather than duplicating?

Reply:
I believe this is the standard way Django records choices in migrations, even when the choices are expressed in a class. The reason is that if the migration referenced a variable in the models file, then if the choices list ever changed there, the migration might stop working.

Example:
https://github.com/openlibhums/janeway/blob/bc8a3279b15d2a88d6390c4136655fcd83056ca4/src/journal/migrations/0031_issue_old_issue_type.py#L35